include/dlaf/blas/tile.h

```diff
@@ -34,10 +34,10 @@
 #define DLAF_GET_ROCBLAS_WORKSPACE(f)                                                                 \
   [&]() {                                                                                             \
     std::size_t workspace_size;                                                                       \
-    DLAF_GPUBLAS_CHECK_ERROR(                                                                         \
+    DLAF_ROCBLAS_WORKSPACE_CHECK_ERROR(                                                               \
```

Comment:
I _think_ `rocblas_{start,stop}_device_memory_size_query` should not return any of the "special" statuses (i.e. `rocblas_status_size_increased`, `rocblas_status_size_unchanged`, or `rocblas_status_continue`) so you should be able to use the regular error checking macro here. Is that the case?